vignettes/config_yml.Rmd

```diff
@@ -0,0 +1,479 @@
+
+```yaml
+    sheet_abcd: "Company Indicators - PACTA Comp"
+```
+
+### project_parameters:
+
+A full example `project_parameters`{.yaml} section might look like:
+
+```yaml
+  project_parameters:
+    scenario_source: "weo_2022"
+    scenario_select: "nze_2050"
+    region_select: "global"
+    start_year: 2022
+    time_frame: 5
+    benchmark_regions_select: "global,european union"
+```
+
+#### scenario_source
+
+`scenario_source`{.yaml} is an identifier of the scenario source to be used. It must be a single string/character value, and it must refer to a valid, accessible, scenario source identifier contained in the scenario data file/s defined by `filename_scenario_tms`{.yaml} and `filename_scenario_sda`{.yaml}. Valid values typically look like `"weo_2023"`{.yaml} or `"geco_2022"`{.yaml}. As an example:
+
+```yaml
+    scenario_source: "weo_2022"
+```
+
+#### scenario_select
+
+`scenario_select`{.yaml} is an identifier of the scenario to be used. It must be a single string/character value, and it must refer to a valid, accessible, scenario identifier contained in the scenario data file/s defined by `filename_scenario_tms`{.yaml} and `filename_scenario_sda`{.yaml}. Valid values typically look like `"nze_2050"`{.yaml}, `"aps"`{.yaml} or `"steps"`{.yaml}. As an example:
+
+```yaml
+    scenario_select: "nze_2050"
+```
+
+#### region_select
+
+`region_select`{.yaml} is an identifier of the region to be used. It must be a single string/character value, and it must refer to a valid, accessible, region identifier contained in the [`r2dii.data::region_isos`{.r}](https://rmi-pacta.github.io/r2dii.data/reference/region_isos_demo.html) datatset. Valid values typically look like `"global"`{.yaml} or `"advanced economies"`{.yaml}. As an example:
+
+```yaml
+    region_select: "global"
+```
+
+#### start_year
+
+`start_year`{.yaml} is the start year of the analysis. Normally, the start year should correspond with year of the publication of the scenario in use. It must be a single numeric value, and it must refer to a valid, accessible, year contained in the scenario data file/s defined by `filename_scenario_tms`{.yaml} and `filename_scenario_sda`{.yaml}. Valid values typically look like `2022`{.yaml} or `2023`{.yaml} (note that this value should *not* be wrapped in quotes). As an example:
+
+```yaml
+    start_year: 2022
+```
+
+#### time_frame
+
+`time_frame`{.yaml} is the number of years (starting from the `start_year`{.yaml}) that the analysis covers, defining the time frame. It must be a single numeric value, and it must define a valid, accessible, time frame  covered by the scenario data file/s defined by `filename_scenario_tms`{.yaml} and `filename_scenario_sda`{.yaml}. Valid values typically look like `5`{.yaml} or `6`{.yaml} (note that this value should *not* be wrapped in quotes). As an example:
+
+```yaml
+    time_frame: 5
+```
+
+#### benchmark_regions_select
```

Comment:
This is not used anymore. while it may be a good idea to think about how to reintroduce some benchmark, right now it is not calculated, because of how much of a drain it was on calculation times

Reply:
completely removed in [da626f9](https://github.com/RMI-PACTA/workflow.multi.loanbook/pull/83/commits/da626f97faf49aa80a94ed466be2e5525d041433)